docs/guides/element-builder/tips.md

```diff
@@ -0,0 +1,66 @@
+---
+heading: Element Builder
+seo: Tips | Element Builder | Cloud Elements API Docs
+title: Tips
+description: Element Builder API Documentation.
+layout: sidebarleft
+apis: API Docs
+platform: elementsbuilder
+breadcrumbs: /docs/guides/home.html
+parent: Back to Guides
+order: 50
+sitemap: false
+---
+# Tips
+
+Tips is a section to find all of your tips and tricks for building high quality elements! *More to come!*
+
+Have a tech tip suggestion? Please contact [Support](mailto:support@cloud-elements.com).
+
+### **Pagination Max**
+
+When building an element in Element Builder, you set up a max pagination, (*number 10 pictured below*) for your element. Cloud Elements uses this to discern what to send to the vendor as a default in the __pageSize__ parameter when making GET list request.
```

Comment:
Make sure that we're consistently referring to the same thing: "Pagination Max", "max pagination", "pageSize" and "Max Page Size" (in the UI) are all referring to the same thing. Can we go with what is in the UI and the name of the parameter? So the Heading would be Max Page Size. The first sentence would be "..., you set up Max Page Size"

Also, regarding the numbered image. I think it'd be a lot clearer if only Max Page Size had a rectangle around it. The numbers are overwhelming.

And a couple of small edits: "...:pencil2:~~__pageSize__~~ `pageSize` parameter when making :pencil2:a (add "a" :point_left: **or** make it plural: "add 'GET list` requests")* `GET list` (:question:should this be `GET /list`?) request.